src/pages/docs/integrations.js

```diff
@@ -10,6 +10,7 @@ import customPluginsLogoImagePath from '../../../content/assets/logos/code/code.
 const INTEGRATION_TYPE_FILTERS = Object.freeze({
```

Comment:
I feel like these categories should be "Catalog Source", "Frontend Plugin" and "Search".